t/t2801-top-cmd.t

```diff
@@ -88,7 +88,7 @@ test_expect_success NO_CHAIN_LINT 'flux-top quits on q keypress' '
 '
 test_expect_success NO_CHAIN_LINT 'flux-top can call itself recursively' '
 	flux jobs &&
-	flux proxy $(cat jobid2) flux jobs -c1 -no {id} >expected.id &&
+	flux proxy $(cat jobid2) flux jobs -c1 -no "\{id\}" >expected.id &&
```

Comment:
Just curious what about your environment caused the quoting to be necessary?

It might be nice to include that in the commit description, since it isn't clear when `{id}` would be expanded to something other than `{id}` by a POSIX shell or even bash.

Reply:
I had just assumed we quoted the format string in most tests. Hmmm.

Reply:
ohh i'm guessing its b/c we're passing the command to flux proxy, vs just running `flux jobs` itself.

Reply:
I can't get it to fail though.. ah, maybe the shell invoked by flux-proxy on your machine is not POSIX sh?

Reply:
Maybe another good "fix" for the tests where `flux proxy` is used would be to `export SHELL=/bin/sh`?

Reply:
yeah, i default to tcsh.  lets go with `/bin/sh` as the goto instead.